From 4d59cc736809a56976e6b9f7c28c9ae2a0d0e478 Mon Sep 17 00:00:00 2001
From: yangjq <yangjq@lenovo.com>
Date: Thu, 17 Dec 2015 20:01:53 +0800
Subject: [PATCH 565/959] IKKRNBSP-3578: motoconfig: Add CONFIG_SUSPEND_DEBUG
 for snapshot of gpio

Add system node /sys/private/tlmm_before_sleep etc.
Used to check and change the gpio before suspend.
This won't save and change gpio's config only after a read of the node

Change-Id: I91da5d6bf7355947aac9f357c53bcf226566de40
Signed-off-by: yangjq <yangjq@lenovo.com>
Reviewed-on: http://gerrit.mot.com/808092
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 31d5b192d69..30cf992cc6d 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -56,3 +56,4 @@ CONFIG_STMVL53L0=y
 CONFIG_EPL8802=y
 CONFIG_SENSORS_FPC_1020=y
 CONFIG_MMC_FFU=y
+CONFIG_SUSPEND_DEBUG=y
-- 
2.11.0

